EasyInventory/src/main/resources/application.properties

```diff
@@ -6,11 +6,8 @@ spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
 spring.data.jpa.repositories.enabled=true
 #We plan on using a cloud database but currently having issues getting it integrated.
 #using local currently
-spring.datasource.url=
-spring.datasource.username=
-spring.datasource.password=
 spring.datasource.driverClassName=com.mysql.jdbc.Driver
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
-#spring.datasource.url=jdbc:mysql://localhost:3306/easyinventory
-#spring.datasource.username=root
-#spring.datasource.password=1234
+spring.datasource.url=jdbc:mysql://localhost:3306/easyinventory
```

Comment:
This wasn't a code review recommendation, I was just trying to debug why the program was giving me database errors.

Reply:
If this is local to your environment, don't commit and push to version control.
Actually... this is a good use for application.properties files that are environment-specific.  :)